### PR TITLE
BrowserFetcher in Chromium

### PR DESCRIPTION
**Pull request by kblok**
progress on #68

### CODE REVIEW

File: src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+using System.IO;
+using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 
 namespace PlaywrightSharp.Chromium
 {
     /// <inheritdoc cref="IBrowserType"/>
     public class ChromiumBrowserType : IBrowserType
     {
+        private readonly string _revision = "733125";
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs]:
```suggestion
        private static readonly string _revision = "733125";
```

File: src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs [hunk 2]
```diff
@@ -25,9 +38,49 @@ public Task<IBrowser> ConnectAsync(ConnectOptions options = null)
...
+
+            var path = options?.Path ?? Path.Combine(Directory.GetCurrentDirectory(), ".local-chromium");
+            string host = options?.Host ?? "https://storage.googleapis.com";
+            var platform = options.Platform ?? GetPlatform();
+
+            Func<Platform, string, BrowserFetcherConfig> paramsGetter = (platform, revision) =>
+            {
+                var archiveName = string.Empty;
+                var executablePath = string.Empty;
+                if (platform == Platform.Linux)
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs]: could we use here the new C# 8 switch expression?

File: src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs (hunk 1 above)
Comment (kblok) [reply src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs]: Playwright has that as an instance prop but I think we can make it static due to our code nature.

File: src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs (hunk 2 above)
Comment (kblok) [reply src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs]: Converted to a switch, but I have many lines to turn them into expressions.

**Pull request merged**